### ISSUE TITLE
Issues with OGG and ODI on Big Data Lite 4.9

### ISSUE THREAD

**Issue by BASoapbox** [oracle/big-data-lite#19]
Greetings!

After downloading  BDL 4.9 I noticed a couple issues and wanted to know whether anyone ha encountered any of them:

1. ODI won't start.  When I click on the ODI icon in the menu bar, it doesn't work.  I do however get it to work (every now and then) if I double click on icon.
2. While going through the Tame Big Data using ODI HOL (
 [http://www.oracle.com/webfolder/technetwork/tutorials/obe/fmw/odi/odi_12c/DI_BDL_Guide/BigDataIntegration_Demo.html](url)) I noticed that records inserted into the Oracle Movie table weren't being inserted into the Hive table.

Anyone noticed any of these issues?  Any suggestions?

**Comment by docolvin** [comment]
Yes, out of the box ODI won't start.  There'a configuration issue with config files in the ~/.odi subdirectory, that I'm still trying to figure out.  You could remove the directory, then ODI would start, but then you lose the connection to the repository.  I'm wondering if there's an issue with the JDK version maybe?

**Comment by syedaliabbas110** [comment]
Hi Docolvin,

Any luck with ODI issue on 4.9 BDAlite ? Its still not coming up..

Thanks

**Comment by syedaliabbas110** [comment]
HI Docolvin

Any luck with ODI on BDAlite4.9 ? Its not coming up...

Thanks
